### DISCUSSION TITLE
乐

### DISCUSSION THREAD

**Discussion by Miuzarte** [Tsuk1ko/CQ-picfinder-robot#311] [一般讨论]
![Stitch_20220518_100938.png](https://user-images.githubusercontent.com/66856838/168942747-f331d6ff-6868-4da4-a278-2e07534a0869.png)
![-35596fd7398d9e93.png](https://user-images.githubusercontent.com/66856838/168942782-e195b43a-407d-4cce-9e78-21e7586aec73.png)
![-3907f0f9ef16ea6.png](https://user-images.githubusercontent.com/66856838/168942801-3252e4d9-eae2-4d94-8ca8-9989fc8621a5.png)

**Comment by Tsuk1ko** [comment]
👀看来太出名也是个坏事

![image](https://user-images.githubusercontent.com/24877906/168947516-47bdfe9e-3e16-4f77-9b4b-4d786dda2c72.png)
